### PR TITLE
Explore int grp difference

### LINKED ISSUE #1002: In Explorer Mode: Show Group/Iteration or difference between 2 groups/iterations
Users want to be able to explore specific groups in the image explorer
They also want to view the difference between 2 groups

Allow user to select integration and group number
If currently selected selection has multiple arrays
      - selection button to find difference
      - also allow the user to select a second integration and group number to be subtracted from first
When apply is selected, updated the image with the new integration/group number data

### PR DESCRIPTION
**Pull request by BradleySappington**
Add ability to show integration and group specific images in explorer mode

### CODE REVIEW

**Comment by BradleySappington** [comment]
Initial front end work, no functionality at this point

**Comment by BradleySappington** [comment]
@mfixstsci @bhilbert4 - Tested and Ready For Review

File: jwql/utils/interactive_preview_image.py [hunk 1]
```diff
@@ -79,6 +79,12 @@ def __init__(self, filename, low_lim=None, high_lim=None, scaling='log', contras
         self.show = show
         self.save_html = save_html
 
+        # Allow sending in of None without overriding defaults
```
Comment (mfixstsci) [inline jwql/utils/interactive_preview_image.py:82]: Good catch here.

File: jwql/utils/interactive_preview_image.py [hunk 2]
```diff
@@ -198,7 +204,7 @@ def get_data(self):
 
         # If a difference image is requested, create the difference image here
         if len(self.data.shape) == 3 and (isinstance(self.group, list) or isinstance(self.integ, list)):
-            diff_img = self.data[0, :, :]*1. - self.data[1, :, :]
+            diff_img = self.data[0, :, :] * 1. - self.data[1, :, :]
```
Comment (mfixstsci) [inline jwql/utils/interactive_preview_image.py:207]: I could be wrong, but the `*` and `/` operators don't require space around the parameters they are operating on 🤔 from a `pep` stand point.

**Comment by BradleySappington** [comment]
INTS		GROUPS		LINK
0		0			https://jwql-dev.stsci.edu/NIRSpec/jw02736005001_02101_00001_nrs2_rate/explore_image/
0		1
0		M
1		0			https://jwql-dev.stsci.edu/NIRSpec/jw02736005001_02101_00001_nrs2_rateints/explore_image/
1		1
1		M			https://jwql-dev.stsci.edu/MIRI/jw01328015001_02105_00004_mirifushort_uncal/explore_image/
M		0			https://jwql-dev.stsci.edu/MIRI/jw01328015001_02105_00004_mirimage_rateints/explore_image/
M		1
M		M			https://jwql-dev.stsci.edu/MIRI/jw01328015001_02105_00004_mirimage_uncal/explore_image/

**Review by mfixstsci — APPROVED** [review]
Hey @BradleySappington, I tested and looked at the code and it looks great. Thanks for submitting this!

File: jwql/utils/interactive_preview_image.py (hunk 2 above)
Comment (BradleySappington) [reply jwql/utils/interactive_preview_image.py:207]: It was old code that PEP yelled at me about:
E226 missing whitespace around arithmetic operator

**Pull request merged**